### PR TITLE
update for google cloud platform source for additional step for creat…

### PR DESCRIPTION
**Pull request by sumoanema**
…ing log sync

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by himanshu219 — APPROVED** [review]

File: docs/send-data/hosted-collectors/google-source/google-cloud-platform-source.md [hunk 1]
```diff
@@ -43,12 +43,23 @@ This Source will be a Google Pub/Sub-only Source, which means that it will only
...
+You need to configure a Pub/Sub Topic in GCP and add a subscription to the Source URL that belongs to the Sumo Logic Google Cloud Platform Source you created.
 
 1. Create a Pub/Sub Topic in GCP. See [Google Cloud documentation](https://cloud.google.com/pubsub/docs/admin#creating_a_topic) for the latest configuration steps.
 1. Create a Pub/Sub subscription to the Source URL that belongs to the Sumo Logic Google Cloud Platform Source you created. See [Google Cloud documentation](https://cloud.google.com/pubsub/docs/admin#creating_subscriptions) for the latest configuration steps.
    * Use a **PushDelivery Method** to the Sumo Logic Source URL. To determine the URL, navigate to the Source on the **Collection** page in Sumo Logic and click **Show URL**. 
 
+:::note
+Warning: Pub/subs are subject to Google Cloud [quotas and limits](https://cloud.google.com/pubsub/quotas#quotas). Evaluate the number of logs against these limitations. If they are higher then the limit/quota, Sumo Logic recommends you split your logs over several topics.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/google-source/google-cloud-platform-source.md:53]:
```suggestion
:::caution
Pub/subs are subject to Google Cloud [quotas and limits](https://cloud.google.com/pubsub/quotas#quotas). Evaluate the number of logs against these limitations. If they are higher then the limit/quota, Sumo Logic recommends you split your logs over several topics.
```

Comment (jpipkin1) [reply docs/send-data/hosted-collectors/google-source/google-cloud-platform-source.md:53]: I recommend you change the note to a caution, and remove "Warning". [See our Style Guide](https://help.sumologic.com/docs/contributing/style-guide/#admonitions) for different types of notes.

File: docs/send-data/hosted-collectors/google-source/google-cloud-platform-source.md [hunk 2]
```diff
@@ -43,12 +43,23 @@ This Source will be a Google Pub/Sub-only Source, which means that it will only
...
+:::
+
+## Exporting Logs from Google Cloud Logging to Pub/Sub
+Once you configure the Pub/Sub, you can export data from Google Cloud Logging to Pub/Sub (created in previous step) using Log Router Sinks. 
+
+Refer [this](https://cloud.google.com/logging/docs/export/configure_export_v2#before-you-begin)  for the permission required for creating Sinks.
+
+Follow [these](https://cloud.google.com/logging/docs/export/configure_export_v2#creating_sink) steps to create log sink to start sending logs to Sumo Logic via Pub/Sub. Note that for **Sink destination** select Pub/Sub topic which you created in the previous step and for **Choose logs to include in sink** you can give the query to include the logs which are needed.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/google-source/google-cloud-platform-source.md:61]:
```suggestion
Once you configure the Pub/Sub, you can export data from Google Cloud Logging to Pub/Sub (created in the previous step) using Log Router Sinks. 

Refer to [Google Cloud documentation](https://cloud.google.com/logging/docs/export/configure_export_v2#before-you-begin) for the permission required for creating Sinks.

Follow the [steps to create a log sink](https://cloud.google.com/logging/docs/export/configure_export_v2#creating_sink) to start sending logs to Sumo Logic via Pub/Sub. Note that for **Sink destination**, select the Pub/Sub topic which you created in the previous step. For **Choose logs to include in sink**, you can give the query to include the logs which are needed.
```

File: docs/send-data/hosted-collectors/google-source/google-cloud-platform-source.md [hunk 3]
```diff
@@ -43,12 +43,23 @@ This Source will be a Google Pub/Sub-only Source, which means that it will only
 
 ## Configure a Pub/Sub Topic for GCP
 
-You need to configure a Pub/Sub Topic in GCP and add a subscription to the Source URL that belongs to the Sumo Logic Google Cloud Platform Source you created. Once you configure the Pub/Sub, you can export data from Google Logging to the Pub/Sub. For example, you can export Google App Engine logs, as described on Collect Logs for Google App Engine.
+You need to configure a Pub/Sub Topic in GCP and add a subscription to the Source URL that belongs to the Sumo Logic Google Cloud Platform Source you created.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/google-source/google-cloud-platform-source.md:46]:
```suggestion
You'll need to configure a Pub/Sub Topic in GCP and add a subscription to the Source URL that belongs to the Sumo Logic Google Cloud Platform Source you created.
```

**Review by kimsauce — COMMENTED** [review]
approved pending @jpipkin1 's changes

**Comment by jpipkin1** [comment]
@sumoanema, I went ahead and updated the article with my suggestions

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**